### PR TITLE
Update Phpdoc and fix array syntax inconsistencies

### PR DESCRIPTION
**Pull request by hacfi**
Going through the code I noticed some inconsistencies and fixed them.

### CODE REVIEW

File: src/Docker/Exception/ImageNotFoundException.php [hunk 1]
```diff
@@ -16,6 +16,6 @@ class ImageNotFoundException extends BaseException
      */
     public function __construct($imageId, BaseException $previous = null)
     {
-        parent::__construct(sprintf('Image not found: "'.$imageId.'"'), 404, $previous);
+        parent::__construct(sprintf('Image not found: "%s"', $imageId), 404, $previous);
```
Comment (hacfi) [inline src/Docker/Exception/ImageNotFoundException.php]: Didn’t make much sense to use sprintf but then concat the string.

Comment (ubermuda) [inline src/Docker/Exception/ImageNotFoundException.php]: Indeed!

**Comment by ubermuda** [comment]
Thanks!

**Comment by hacfi** [comment]
@ubermuda Thanks for the quick merge..love this library & will definitely contribute more in the future!

**Pull request merged**